test/api/query.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { QueryOperation, QueryOperator } from '../../src/lib/types';
+import { stackInstance } from '../utils/stack-instance';
+import { TEntries } from './types';
+
+const stack = stackInstance();
+describe('Query API tests', () => {
+  it('should add a where filter to the query parameters', async () => {
+    const query = await makeQuery('blog_post')
+      .where('title', QueryOperation.EQUALS, 'The future of business with AI')
+      .find<TEntries>();
+    expect(query.entries[0].title).toEqual('The future of business with AI');
+  });
+  it('should add a where filter to the query parameters', async () => {
+    const query = await makeQuery('blog_post').where('_version', QueryOperation.IS_LESS_THAN, 3).find<TEntries>();
+    expect(query.entries[0].title).toEqual('The future of business with AI');
+  });
+  it('should add a where-in filter to the query parameters', async () => {
+    const query = await makeQuery('blog_post')
+      .whereIn('author', makeQuery('author').where('uid', QueryOperation.EQUALS, 'blt09f7d2d46afe6dc6'))
```

Comment:
remove the credentials from the file @nadeem-cs

Reply:
Resolved in [this](https://github.com/contentstack/contentstack-javascript/pull/110) PR.